schedules/delete_schedule.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+
+from temporalio.client import Client
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_schedule_handle(
+        "workflow-schedule-id",
+    )
+    desc_handle = await handle.describe()
+    print(f"{desc_handle.schedule}")
```

Comment:
Why print out the schedule? If we do want to, might as well add a prefix word or something instead of just random dump of data to screen.